pagopa/pn-frontend
# refactor(PN-11896): rework on courtesyAddressContacts

## Short description
Rework of common components and courtesy addresses for PF and PG.

## List of changes proposed in this pull request
- Removed `actions` prop and use of `memo` in DigitalContactsCard component
- Catched error in IOContact component (only PF)
- Moved content of CourtesyContactsList in CourtesyContacts component

## How to test
Login as PF/PG; check in contacts page UX/UI is not changed, but components work correctly.
